public/vendor/plugins/sortt/sortt.js

```diff
@@ -0,0 +1,62 @@
+/*
+  Sortt
+
+  @License MIT
+  @Author 6543
+  @Repository https://gitea.com/6543/gitea_sortt
+  @Version 1.2
+
+*/
+
+
+// test if jQuery is available
+if(!window.jQuery) {
+	console.log("sortt.js: ERROR no jQuery found!");
+} else {
+
+  //use JQuery to bind event on each <th> with a "data-sortt" attribute
+  $('th').each(function() {
+    if ($(this)[0].dataset.sortt) $(this).on('click', function(e) {
+	    var data = $(this)[0].dataset.sortt
```

Comment:
shouldn't there be a semi-colon in the end of line? I know that FF / Chrome may ignore that, but IE (as I remember) won't.

Reply:
😅